environment-setup/ansible/TP2/README.md

```diff
@@ -0,0 +1,280 @@
+# Starting the cluster configuration with ansible
+Currently all the configuration, package installation is done via ansible, the playbook-tp2.yml contains the multiple roles that are used to achieve final cluster configuration. It will run:
+
+1. Install Packages and do initial user creation (admins and students)
+2. The monitoring-service role creates an instance of redis and starts the monitoring service as part of the TP3.
+3. Installs docker, k3s and does general kubernetes configurations.
+4. Creates all the kubernetes service users and setup their roles and permissions.
+5. Permissions were updated to allow users to only create pvcs to a specific storage class and within that class to only create one pvc and disallow all others. (new)
+5. Installs rancher and exposes it on port 30443, creates all users in rancher. (new)
+6. Installs argocd and exposes it on port 31443 and creates all users in argocd.
+7. Setup the firewall denying all requests from the outside world except for the SSH Port. (new)
+
+After running the ansible playbook in the cluster (namely playbook-tp2.yml) all of the above will be performed. Keep an eye out for the logs that it throws as the administrator password for rancher and ansible will be printed out. If you missed that, you can also use rancher's command in the cluster vm fetch such password. It's random but it's deterministic so it shouldn't change through runs for the same host machine.
```

Comment:
Wow, can we please also write down the Rancher admin password to a file? This way it doesn't get lost in the logs. What is the command we need to run to get the password from the command line?

Reply:
Please mention that the command is provided below; I was left hanging there...